src/commonMain/kotlin/dev/zacsweers/fieldspottr/Home.kt

```diff
@@ -216,27 +220,27 @@ fun Home(state: HomeScreen.State, modifier: Modifier = Modifier) {
         },
       )
     },
-    floatingActionButton = {
-      DateSelector(state.date) { newDate -> state.eventSink(FilterDate(newDate)) }
-    },
     snackbarHost = {
       SnackbarHost(hostState = snackbarHostState) { snackbarData ->
         Snackbar(snackbarData = snackbarData)
       }
     },
   ) { innerPadding ->
-    Column(Modifier.padding(innerPadding), verticalArrangement = spacedBy(16.dp)) {
-      GroupSelector(
-        state.selectedGroup,
-        modifier = Modifier.align(CenterHorizontally).padding(horizontal = 16.dp),
-      ) { newGroup ->
+    Column(modifier = Modifier.padding(innerPadding), verticalArrangement = spacedBy(8.dp)) {
+      GroupSelector(state.selectedGroup, modifier = Modifier.align(CenterHorizontally)) { newGroup
+        ->
         state.eventSink(ChangeGroup(newGroup))
       }
 
+      val cornerSlot = movableContentOf {
```

Comment:
A `movableContentOf` that isn't `remember`ed is suspicious here, that could lead to the state loss issue inside `DateSelector` if this re-runs and a new `movableContentOf` is created.

Reply:
oooo that's a good catch!

Reply:
[0e828dd](https://github.com/ZacSweers/FieldSpottr/pull/26/commits/0e828dd3e35586a6833f08400429ae29f1a14309)